pulumi/infra/nomad_job.py

```diff
@@ -12,8 +13,10 @@
 _ValidNomadVarTypes = pulumi.Input[
     Union[
         _ValidNomadVarTypePrimitives,
-        Mapping[str, _ValidNomadVarTypePrimitives],
-        NomadServicePostgresDbArgs,  # Upsettingly, this is a Mapping[str, object]
+        Mapping[str, pulumi.Input[_ValidNomadVarTypePrimitives]],
+        # Upsettingly, TypedDicts are a Mapping[str, object]
+        NomadServicePostgresDbArgs,
```

Comment:
Currently we have N NomadServicePostgresDbArgs arguments to grapl-core. I may change this to a Mapping[service -> db args] in the near future.